src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -48,6 +48,7 @@ class DeployTask extends AbstractServerTask {
 
     protected ApplicationXmlDocument applicationXml = new ApplicationXmlDocument();
     private static final boolean DEFAULT_CONTAINER = false;
+    private static final String PROJECT_ROOT_BUILD_LIBS = "build/libs";
```

Comment:
This just occurred to me (and it may apply to maven as well), but shouldn't we use the project build dir instead of hardcoding "build/"? For gradle that is `project.buildDir`.

Reply:
Changed to use buildDir.  For cases where buildDir is outside of the project root, that will be handled separately in https://github.com/OpenLiberty/ci.maven/issues/876 , but for now it will just use `build/libs` if that happens since it must be within the project root that is mounted.

Reply:
Don't you need to open a gradle issue also to handle buildDir outside project root?

Reply:
Opened https://github.com/OpenLiberty/ci.gradle/issues/489